js/analytics_text.js

```diff
@@ -0,0 +1,25 @@
+
+$(function(){
+	// get the name of the current page
+	var p = window.location.pathname.split("/");
+	var filename = p[p.length - 1];
```

Comment:
Maybe `filename` could be `resource`? Also, is there only one `analytics`? Are there e.g. account-level analytics resources?